CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 We always appreciate your help in testing open pull requests by deploying custom builds of actions-runner-controller onto your own environment, so that we are extra sure we didn't break anything.
 
-It is especially true when the pull request is about GitHub Enterprise, both GHEC and GHES, as [maintainers don't have GitHub Enterprise environments for testing](/README.md#github-enterprise-support).
+It is especially true when the pull request is about GitHub Enterprise, both GHEC and GHES, as [maintainers don't have GitHub Enterprise environments for testing](https://github.com/actions-runner-controller/actions-runner-controller/blob/master/docs/detailed-docs.md#github-enterprise-support).
```

Comment:
```suggestion
It is especially true when the pull request is about GitHub Enterprise, both GHEC and GHES, as [maintainers don't have GitHub Enterprise environments for testing](docs/detailed-docs.md#github-enterprise-support).
```

Good catch! But should this be written as a relative path so that this markdown is viewable and testable in forks too?

Reply:
@mumoshu  I wasn't sure about the convention since I saw a mix of relative and absolute paths.  I will change these to relative paths. 👍

Reply:
Oops, sorry! I just committed the suggestion to show us if it works or not. If it turned out to work, would you mind fixing other links in the same way, too?

Reply:
Nice- relative paths seem to work
https://github.com/actions-runner-controller/actions-runner-controller/blob/1c762da6a1f693de0e021df0e44b283f8ce2a358/CONTRIBUTING.md